src/app/components/card/card.component.html

```diff
@@ -1,56 +1,50 @@
+                <mat-icon>groups</mat-icon>
+                <div class="text-ellipsis">
+                    {{card?.organisation}}
+                </div>
 
-    <div class="preview" [class.no-preview]="card?.preview === 'assets/no_preview.png'">
-        <div class="highlights" *ngIf="card?.tabs.length > 0">
-            <div class="section_title">{{ 'Highlights' | translate}}</div>
-            <div *ngFor="let tab of card?.tabs" class="highlight">
-                ⇾ {{tab}}
             </div>
         </div>
-        <div class="highlights" *ngIf="card?.tabs.length === 0">
-            <div class="section_title">{{ 'No Highlights' | translate}}</div>
-        </div>
+        <div class="preview_holder"></div>
         <ng-container *ngIf="card?.preview$ | async | json_to_preview; let preview; else notPreview">
-            <img src="{{preview}}" class="image" />
+            <img src="{{preview}}" class="preview_image" />
         </ng-container>
         <ng-template #notPreview>
-            <img src="assets/no_preview.png" class="image" />
+            <img src="assets/no_preview.png" class="preview_image" />
         </ng-template>
     </div>
-    <div class="groups">
-        <div *ngIf="!!card.organisation && card.organisation !== NO_ORGANISATION && publicOrg">
-            <div class="section_title">{{ 'Organisation' | translate}}</div>
-            <mat-chip-list>
-                <mat-chip class="organisation" >{{card?.organisation}}</mat-chip>
-            </mat-chip-list>
-        </div>
-        <div class="readers">
-            <div class="section_title">{{ 'Readers' | translate}}</div>
-            <mat-chip-list>
-                <mat-chip *ngIf="card?.readers.length === 0" class="group"> -- </mat-chip>
-                <mat-chip *ngFor="let reader of readers" class="group" [class.mygroup]="reader.in">{{reader.name}}</mat-chip>
-            </mat-chip-list>
+    <mat-card-content class="card_meta">
+        <div class="meta_title-container">
+            <div class="meta_title text-ellipsis" matTooltip="{{'View' | translate}}&nbsp;{{card?.title}}" [matTooltipPosition]="'above'"> {{card?.title}}</div>
+            <arlas-config-menu #configMenu class="config-menu" matTooltip="{{'Menu' | translate}}" [zone]="'config.json'"
+                               [actions]="card?.actions" (actionExecutedEmitter)="afterAction($event)"></arlas-config-menu>
         </div>
-        <div class="writers">
-            <div class="section_title">{{ 'Writers' | translate}}</div>
-            <mat-chip-list>
-                <mat-chip *ngIf="card?.writers.length === 0" class="group"> -- </mat-chip>
-                <mat-chip *ngFor="let writer of writers" class="group" [class.mygroup]="writer.in">{{writer.name}}</mat-chip>
-            </mat-chip-list>
-        </div>
-    </div>
-
+        <mat-card-subtitle class="meta_subtitle">{{ 'Last update' | translate }} : {{card?.last_update_date | date}}</mat-card-subtitle>
```

Comment:
We should use the i18n variables here, the spacing around ':' depends on the language (space before in French but none in english and spanish)